### ISSUE TITLE
Mac version lacks icon and title(?)

### ISSUE THREAD

**Issue by Jmuccigr** [colmap/colmap#16]
Just started playing with this and I noticed right away that the Mac binary has the generic icon. Not a big deal, but even something basic would be an improvement and add to the project's appearance.

I also noticed when I tried to force-quit something that Colmap doesn't have a title in that list of running applications that appears when you cmd-opt-esc. I have no idea why, but I assume something is missing somewhere. :-)

**Comment by ahojnnes** [comment]
Thanks for the report and trying it out. I'll look into it in 2 weeks as I
am traveling at the moment.

On 03 Jul 2016, at 07:51, John Muccigrosso <notifications@github.com> wrote:

Just started playing with this and I noticed right away that the Mac binary
has the generic icon. Not a big deal, but even something basic would be an
improvement and add to the project's appearance.

I also noticed when I tried to force-quit something that Colmap doesn't
have a title in that list of running applications that appears when you
cmd-opt-esc. I have no idea why, but I assume something is missing
somewhere. :-)

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/colmap/colmap/issues/16>, or mute the thread
<https://github.com/notifications/unsubscribe/AAIMo9hJKGjQ8e6QelnZqRzotzcsRf7qks5qR755gaJpZM4JD34s>
.
